ckanext/dia/commands.py

```diff
@@ -125,6 +161,7 @@ def _get_datastore_table_page(self, context, offset=0):
                 print "Resource '%s' found" % record['name']
             except logic.NotFound:
                 resource_id_list.append(record['name'])
+                context.pop('__auth_audit', None)
```

Comment:
This fixes the issue that was causing the KeyError to be thrown below, and causing a bunch of false errors in the output. `resource_show` checks whether the calling user is authorized, but doesn't clean up after itself when the action returns a NotFound, so I'm doing it manually here.